ClientLib/src/main/java/com/o3dr/services/android/lib/drone/mission/item/command/Takeoff.java

```diff
@@ -38,11 +40,13 @@ public void setTakeoffAltitude(double takeoffAltitude) {
     public void writeToParcel(Parcel dest, int flags) {
         super.writeToParcel(dest, flags);
         dest.writeDouble(this.takeoffAltitude);
+        dest.writeDouble(this.getMinPitch());
```

Comment:
for variables inside your class it's more efficient to use a direct reference instead of a getter.